app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -5,12 +5,14 @@
 import android.support.design.widget.NavigationView;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.GravityCompat;
```

Comment:
Make this a static import since you're using it just for the `START` static value of the class. This should read as: `import static android.support.v4.view.GravityCompat.START;`